tutorials/fundamentals/typeof-number.md

```diff
@@ -0,0 +1,26 @@
+`typeof` is a useful operator that returns, in a string,
+the `typeof` the variable. When using this operator with
```

Comment:
"the [primitive type](https://masteringjs.io/tutorials/fundamentals/primitives) of the given variable."